src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/courtsentencing/CourtSentencingMappingResource.kt

```diff
@@ -105,6 +105,85 @@ class CourtSentencingMappingResource(private val mappingService: CourtSentencing
       )
     }
 
```

Comment:
will probably split this resource as have done for the int tests as Court Sentencing is a large domain